Mathlib/AlgebraicGeometry/Scheme.lean

```diff
@@ -147,6 +147,11 @@ lemma appLE_congr (e : V ≤ f ⁻¹ᵁ U) (e₁ : U = U') (e₂ : V = V')
 noncomputable def homeomorph [IsIso f] : X ≃ₜ Y :=
   TopCat.homeoOfIso (asIso <| f.val.base)
 
+/-- A morphism of schemes `f : X ⟶ Y` induces a local ring homomorphis from `Y.presheaf.stalk (f x)`
+to `X.presheaf.stalk x` for any `x : X`. -/
+def stalkMap (x : X) : Y.presheaf.stalk (f.val.base x) ⟶ X.presheaf.stalk x :=
```

Comment:
If you want access to the dot notation, can this just be an `abbrev`?

Reply:
I thought the principle is to hide the implementation detail that schemes are implemented as presheafed spaces as much as possible, so we mostly mirror the API. Do you think performance is better if this is an `abbrev`?